region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -755,6 +755,43 @@
       "note": "Come in with a spring ball bounce after gaining speed on a remote runway using at least 34 tiles.",
       "devNote": "This requires extra run speed of at least $6.0."
     },
+    {
+      "link": [1, 3],
+      "name": "Use Flash Suit",
```

Comment:
These flash suit uses assume wave, essentially.  But you can hit the gate with a super even with a flash suit using springball or a quick jump morph.
The water entry shouldn't always require hijump either.